core/src/main/java/hudson/Functions.java

```diff
@@ -1462,36 +1462,6 @@ public static int determineRows(String s) {
         return Math.max(5,LINE_END.split(s).length);
     }
 
-    /**
-     * Converts the Hudson build status to CruiseControl build status,
-     * which is either Success, Failure, Exception, or Unknown.
-     */
-    public static String toCCStatus(Item i) {
```

Comment:
Formally breaks the binary compatibility.
I would just keep the Deprecated Restricted Sub method, which returns "unknown"